EmpleoDotNet/Models/EmpleadoContext.cs

```diff
@@ -22,8 +22,12 @@ public EmpleadoContext()
         public DbSet<JobOpportunity> JobOpportunities { get; set; }
         public DbSet<Location> Locations { get; set; } 
         public DbSet<Tag> Tags { get; set; }
+        public DbSet<IdentityUserRole> UserRoles { get; set; }
```

Comment:
Pensaba que ya Claudio habia agregado el soporte para Identity, esto es parte de esos cambios? o los agregaste de nuevo en tu branch y no estaba up to date?